spec/models/ingredient_mention_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe IngredientMention, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You have to make sure that you test the relationships of your new model to make sure they work. Likewise you should probably update both other model specs to ensure they can utilise this new join model correctly.

Reply:
EXAMPLES OF RELATIONSHIP TESTS IN RSPEC:
```
  describe 'when observing relationships' do
    it 'belongs to a category' do
      expect(subject.category).to eq(Category.first)
    end

    it 'has many purchases' do
      cart = create(:cart)
      create(:purchase, cart: cart, product: subject)
      expect(subject.purchases.count).to eq(1)
    end
  end
```